connector_ecommerce/components/sale_order_onchange.py

```diff
@@ -77,7 +77,7 @@ def play(self, order, order_lines):
         :rtype: dict
         """
         # play onchange on sales order
-        order = self.play_onchanges("sale.order", order, self.order_onchange_fields)
+        order = self.play_onchanges("magento.sale.order", order, self.order_onchange_fields)
```

Comment:
@DarkoNikolovski wondering if fix should be here, because `connector_ecommerce  does not depend on `magento`.

Reply:
@paulius-sladkevicius You are correct, I have missed that. Probably the best option is to override the method in `magento_cooonector` module and update the changes there.

Reply:
Wondering how it was working in the earlier versions, if this `play_onchanges` is new stuff? Can you double check, because overwrite doesn't sound like a right way to do it.

Reply:
@paulius-sladkevicius I have compared the two solutions 8.0 and 13.0, they are similar but the thing is that in Odoo v 8.0 if the field is not presented it is [ignored](https://github.com/odoo/odoo/blob/8.0/openerp/models.py#L5393), while in Odoo v 13.0 there is [validation](https://github.com/odoo/odoo/blob/13.0/odoo/models.py#L5189) if all fields are in the model, and that is where it is failing in v.13

Reply:
I see, but you've comperaed quote old version, I was more interested in how it was working with [v12 magento connector](https://github.com/OCA/connector-magento/tree/12.0/connector_magento) vs v13. Anyway, looking forward to seeing a fix :-)

Reply:
@paulius-sladkevicius Odoo v.12 has the same code as Odoo v.8 that is why it is working for v.12.
The new fix that I have provided is focused in general for the `connector-ecommerce`, since the issue is related directly with the `play_onchanges` method and it will happen with any other similar modules which have extra fields that are not part of the onchange model.